packages/by-name/cli-release/package.nix

```diff
@@ -2,20 +2,16 @@
 # SPDX-License-Identifier: AGPL-3.0-only
 
 {
-  lib,
   contrast,
+  kata,
   microsoft,
-  genpolicy ? microsoft.genpolicy,
 }:
 
 (contrast.overrideAttrs (
   _finalAttrs: previousAttrs: {
-    prePatch = ''
-      install -D ${lib.getExe genpolicy} cli/genpolicy/assets/genpolicy
-      install -D ${contrast.settings}/genpolicy-settings.json cli/genpolicy/assets/genpolicy-settings.json
-      install -D ${contrast.rules}/genpolicy-rules.rego cli/genpolicy/assets/genpolicy-rules.rego
-      # TODO(burgerdev): cli/genpolicy/assets/allow-all.rego is insecure and deliberately omitted
-      install -D ${contrast.embeddedReferenceValues} internal/manifest/assets/reference-values.json
+    postPatch = ''
+      install -D ${microsoft.genpolicy.settings}/genpolicy-settings.json cli/genpolicy/assets/genpolicy-settings.json
```

Comment:
Can we also have a vendor suffix for this file?

Reply:
Good catch